libs/features/mas/docs/common.js

```diff
@@ -1,38 +1,65 @@
-import '../../../deps/custom-elements.js';
-import '../dist/mas.js'; 
-const init = () => {
-  const ENVS = {
-    qa: 'qa-odin',
-    stage: 'stage-odin',
-    prod: 'odin',
-  };
-  const href = window.location.href;
-  const envOverride = new URL(href).searchParams.get('aem.env');
-  const env =
-    envOverride && ENVS[envOverride]
-      ? ENVS[envOverride]
-      : ENVS.prod;
-  const meta = document.createElement('meta');
-  meta.name = 'aem-base-url';
-  meta.content = `https://${env}.adobe.com`;
-  document.head.appendChild(meta);
-  // theme
+const MAS_IO_URL = 'mas-io-url';
+
+let polyfillsPromise;
+export async function polyfills() {
+  if (polyfillsPromise) return polyfillsPromise;
+  let isSupported = false;
+  document.createElement('div', {
+    // eslint-disable-next-line getter-return
+    get is() {
+      isSupported = true;
+    },
+  });
+  if (isSupported) {
+    polyfillsPromise = Promise.resolve();
+  } else {
+    polyfillsPromise = await import('../../../deps/custom-elements.js');
+  }
+  return polyfillsPromise;
+}
+
+const toggleTheme = (theme, params) => {
+  document.body.className = 'spectrum spectrum--medium';
+  document.body.classList.add(`spectrum--${theme}`);
+  document.querySelector('sp-theme')?.setAttribute('color', theme);
+  params.set('theme', theme);
+  history.replaceState(
+      null,
+      '',
+      `${location.pathname}?${params}`,
+  );
+}
+
+const init = async () => {
+  await polyfills();
   const params = new URLSearchParams(document.location.search);
+  if (params.get(MAS_IO_URL)) {
+    const meta = document.createElement('meta');
+    meta.name = MAS_IO_URL;
+    meta.content = params.get(MAS_IO_URL);
+    document.head.appendChild(meta);
+  }
+  
+  // theme
   const darkTheme = params?.get('theme')?.toLowerCase() === 'dark';
   const theme = document.createElement('script');
   theme.setAttribute('src', `../../spectrum-web-components/dist/themes/${darkTheme ? 'dark' : 'light'}.js`);
   theme.setAttribute('type', `module`);
   document.head.appendChild(theme);
   
   // mas-commerce-service
-  const masCommerceService = document.createElement('mas-commerce-service');
-  ['locale','language','env'].forEach((attribute) => {
+  const masCommerceService = document.querySelector('mas-commerce-service');
```

Comment:
most of mas-commerce-service elements i saw in doc files are placed after the script, don't we risk not to find any here (and shouldn't we place the tag before?)